spotify_dl/setup_creds.sh

```diff
@@ -0,0 +1,2 @@
+export SPOTIPY_CLIENT_ID=80caa39428394b249a615e056848b8db
```

Comment:
secrets again

Reply:
i deleted this file. trying to delete it again returns error 404. any clues on resolution . ill be glad to use

Reply:
let me revoke the the secrets maybe